mmdet/datasets/coco.py

```diff
@@ -115,7 +115,7 @@ def parse_data_info(self, raw_data_info: dict) -> Union[dict, List[dict]]:
         if self.data_prefix.get('seg', None):
             seg_map_path = osp.join(
                 self.data_prefix['seg'],
-                img_info['file_name'].replace('jpg', 'png'))
+                img_info['filename'].rsplit('.', 1)[0] + self.seg_map_suffix)
```

Comment:
is the change from file_name to filename correct? Did you verify the modification?